collector/settings.md

```diff
@@ -0,0 +1,499 @@
+---
+title: 'pganalyze Collector settings'
+backlink_href: /docs/collector
+backlink_title: 'pganalyze Collector'
+---
+
+The collector can be configured either through an `INI` config file (in a package-based install) or environment
+variables (typically for running via its Docker image). Most settings can be configured through either mechanism.
+If both are present, the config file takes precedence. Note that a single collector instance can monitor more than
+one database server, though this is not supported when configuring through environment variables.
+
+For the `INI`-based setup, the `[pganalyze]` section describes settings that apply to all servers. Other sections
+describe the servers to monitor, how to connect to them, and server-specific configuration settings. You should
+name the other sections after the servers they correspond to, though note these are not the names that will appear
+in the app. In-app names are based on hostname settings as determined during monitoring. When you set environment
+variables in addition to specifying a configuration file, the environment variable settings apply to all monitored
+servers.
+
+After you make changes, you can run `pganalyze-collector --test --reload` to verify the new configuration and load
+the new configuration in the collector background process if they work correctly. This minimizes monitoring
+interruptions and simplifies config file updates.
+
+
+## Configuration settings
```

Comment:
This heading also shows up in the TOC, which looks a bit odd. I wonder if we can just skip the heading, and make it one large intro section?

Reply:
Yeah, you're right--I think that works better.